lib/hygeia_web/live/auto_tracing_live/clinical.ex

```diff
@@ -272,13 +259,16 @@ defmodule HygeiaWeb.AutoTracingLive.Clinical do
     {start_date, problems} = index_phase_start_date(case, days_before_test: @days_before_test)
```

Comment:
Remove problems from all the statements

```suggestion
    start_date = index_phase_start_date(case, days_before_test: @days_before_test)
```